### PR TITLE
Prevent autosay() from creating/deleting AIs

### LINKED ISSUE #5271: AI creation spam due to announcements
#### Brief description of the issue
Every time an announcement happens, a /mob/living/silicon/ai is spawned to say a line, then deleted. AIs all get hard del()'d because they are mobs, and it takes ~1 second to delete them, thus creating an annoying lagspike whenever it happens.

#### What you expected to happen
Not lag whenever there's an announcement.

#### What actually happened
Lag.

#### Steps to reproduce
Announce.

### PR DESCRIPTION
**Pull request by Arokha**
Now it gets one dedicated AI it can move around instead.

Fixes #5271

### CODE REVIEW

File: code/game/objects/items/devices/radio/radio.dm [hunk 1]
```diff
@@ -247,14 +247,12 @@ var/global/list/default_medbay_channels = list(
 	if (!connection)
 		return
 
-	var/mob/living/silicon/ai/A = new /mob/living/silicon/ai(src, null, null, 1)
+	var/static/mob/living/silicon/ai/announcer/A = new /mob/living/silicon/ai/announcer(src, null, null, 1)
```
Comment (Atermonera) [inline code/game/objects/items/devices/radio/radio.dm]: And the `static` here means the Singleton AI will be found when the proc is called by other objects, or will `new` still make fresh copies?

Comment (Arokha) [reply code/game/objects/items/devices/radio/radio.dm]: The way it appears to work (I was not aware of this before working on this issue, actually) is that declaring a static inside a proc makes it a sort of 'global static' that is only initialized once. I believe it makes it literally a global var, but somehow scoped to just this proc. This line is basically 'skipped' after it's initialized once (in truth, it's initialized when the game starts, this line 'isn't really here). It's a little weird how Byond handles this, but, it works and it's apparently intended to behave like I've described.

Comment (Atermonera) [reply code/game/objects/items/devices/radio/radio.dm]: Ok. Byond reference docs don't describe static, or if they do it's somewhere hidden, but it's not surprising there's support for a Singleton-style object management

Comment (Arokha) [reply code/game/objects/items/devices/radio/radio.dm]: Oh, sorry. `static` is an alias for `global`, maybe I misunderstood your question.
They are the same thing.

**Comment by Anewbe** [comment]
As a question, this announcement AI doesn't show up on any sort of visible interface, a la Law Interface thing?

**Comment by Arokha** [comment]
It should not. For example the silicon law editor admin verb does:
`var/mob/living/silicon/S = input("Select silicon.", "Rename Silicon.") as null|anything in silicon_mob_list`
And this AI removes itself from all those mob lists when it's created.

**Comment by Anewbe** [comment]
Works for me.

**Pull request merged**